tutorials/fundamentals/concat-num-to-string.md

```diff
@@ -0,0 +1,10 @@
+If you're combining elements [you should not use `concat()`](/tutorials/fundamentals/string-concat).
+You should instead use the `+` operator as follows:
+
+```javascript
+let example = 'Hello World';
+
+let demo = example + 42; // 'Hello World42'
```

Comment:
I'd also add an example of adding ' ' so you get 'Hello World 42'.

Also, let's add a note about `console.log(str, 42)`: you don't need to concat if you're just looking to print.